src/saturn_engine/worker/topics/rabbitmq.py

```diff
@@ -98,7 +99,7 @@ async def run(self) -> AsyncGenerator[t.AsyncContextManager[TopicMessage], None]
                 async with queue.iterator() as queue_iter:
                     async for message in queue_iter:
                         attempt = 0
-                        yield self.message_context(message)
+                        yield self.message_context(message, attempt)
```

Comment:
`attempt` is not exactly the attempt on this specific message, maybe it would make sense to track a `LRU` dict that track the attempt per message id instead. fwiw, it's always going to be `0` at this point (look the line above ;) )